science/2022-03-03-science-blog-5/index_de.md

```diff
@@ -0,0 +1,898 @@
+
+Bei Antigen-Schnelltests ist dieses Verhältnis sogar noch überzeugender.
+
+<figure>
+    <img src="./ppa_risk_infection_ct_timeline_rat_de.png" class="figure-img img-fluid" alt="Positivenanteil nach Risikobenachrichtigung im zeitlichen Verlauf (RAT)."/>
+    <figcaption aria-hidden="true">Abb. 5: Positivenanteil nach Risikobenachrichtigung im zeitlichen Verlauf (RAT).</figcaption>
+</figure>
+
+*Die Warnungen durch die CWA sind damit, trotz der bekannten Unzulänglichkeiten der BLE-Technologie, präzise und zuverlässig. Die CWA warnt die Richtigen.*
+
+Die Bedeutung der CWA lässt sich auch im aktuellen Pandemiegeschehen illustrieren. Die Anzahl der bundesweiten Neuinfektionen der letzten 7 Tage lag bei 1.007.987 (Stand: 28. Februar 2022). Im selben Zeitraum wurden 1.388.486 positive Corona-Testergebnisse in das Backend der CWA übermittelt und 330.228 von dort von CWA-Nutzenden abgerufen (23,8%). Von diesen wurden wiederum rund zwei Drittel (70,7%) geteilt. Insgesamt wurden damit knapp 16,8% der positiven Tests in Deutschland innerhalb der CWA genutzt und geteilt, um andere zu warnen (in den letzten 7 Tagen waren dies 233.377 Personen). Unter der Annahme, dass pro warnender Person 23 CWA-Nutzende eine rote Warnung erhalten (dieser Wert lag in den letzten 30 Tagen höher als über den gesamten Zeitraum betrachtet), ergeben sich aktuell ca. 5.367.700 rot Gewarnte, von denen sich etwa 15% testen lassen und ihr Resultat über die CWA registrieren. Zudem kann man davon ausgehen, dass sich weitere 72% testen lassen (dies zum großen Teil auch offiziell, s.&nbsp;o.), ihr Testergebnis aber nicht in der CWA registrieren können. Von diesen insgesamt ca. 3.887.000 werden etwa 14,1% und damit bis zu 548.500 weitere CWA-Nutzende positiv getestet. Infolge der schnellen Warnung und der unmittelbaren Verhaltensanpassungen potenziell infizierter CWA-Nutzender werden Infektionsketten durch die CWA frühzeitig beendet.
+
+Eine genaue Bezifferung der beendeten Infektionsketten bzw. der vermiedenen Fälle ist nur im Rahmen einer Simulation oder Modellierung möglich. Dabei muss man die tatsächliche Situation mit einer virtuellen (hypothetischen) vergleichen, in der es z.&nbsp;B. keine CWA gibt. Dies ist für einen zukünftigen Blog-Beitrag geplant.
+
+Zusammenfassend zeigt die aktuelle Auswertung, dass die CWA eine bedeutende Rolle bei der Eindämmung der Pandemie spielt. Durch die direkte Rückmeldung von Testergebnissen in die CWA und das kurze Zeitfenster zwischen Risikobegegnung und Warnung können umgehend und ohne Einbeziehung Dritter Infektionen aufgedeckt und somit schnellst möglich beendet werden. Am Rande sei erwähnt, dass die frühzeitige Erkennung einer Infektion auch für die betroffene Person selbst vorteilhaft ist, insbesondere wenn diese ein erhöhtes Risiko für einen schweren Verlauf der Erkrankung hat und von einer frühzeitigen gezielten Therapie profitiert.
```

Comment:
"schnellst möglich" => "schnellstmöglich"
https://www.duden.de/rechtschreibung/schnellstmoeglich

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/2500, thanks